rubrics/impl/src/main/java/org/sakaiproject/rubrics/logic/RubricsServiceImpl.java

```diff
@@ -1026,6 +1026,10 @@ public String generateLang(){
 
         return lines.toString();
     }
+    
+    public Boolean isWeightedEnabled(){
+        return serverConfigurationService.getBoolean("rubrics.enable.weighted", true);
```

Comment:
Can this is isWeightedEnabled check to see if the rubric is weighted and if it isn't then check the server config property to see whether to allow it. This has the end result of preserving existing rubrics weighted functionality.

Reply:
I understand your point,
But that's how the current PR works,
The PR only hides the buttons to change if a rubric is weighted

So, if a rubric was weighted, it will remain weighted
That's why I was asking about what we should do with the already weighted rubrics at the DB